client-templates/nexus3/.env.sample

```diff
@@ -20,10 +20,17 @@ BROKER_HEALTHCHECK_PATH=/healthcheck
 
 # Provide RES_BODY_URL_SUB with the URL of the Nexus without credentials and http protocol
 # This URL substitution is required for NPM integration
-# RES_BODY_URL_SUB=http://<your.nexus.hostname>/repository
+# RES_BODY_URL_SUB=https://<your.nexus.hostname>/repository
```

Comment:
is there a specific reason for this change? does `npm-deps` accommodate that?

Reply:
Good point. Just looking at the comment on line 21 it says `http`

Reply:
yea we discovered with customers during private beta this is https for Nexus. It's copied from Artifactory env, where it's a little strange, the tarball urls start http (not https) - some Artifactory bug.

Reply:
I see. So its ok to leave as `https` then and remove that part of the comment on line 21?